.github/workflows/config/spelling_allowlist.txt

```diff
@@ -8,6 +8,7 @@ COBYLA
 CPU
 CPUs
 CUDA
+CUDAQ
```

Comment:
Please do not add this. CUDA Quantum is the name to use in text. If this referred to a code snipped, then please indicate it as such (make it a code block or fenced by backticks).